selfdrive/car/toyota/carcontroller.py

```diff
@@ -26,7 +26,7 @@ def update(self, enabled, active, CS, frame, actuators, pcm_cancel_cmd, hud_aler
              left_line, right_line, lead, left_lane_depart, right_lane_depart):
 
     # gas and brake
-    if CS.CP.enableGasInterceptor and enabled:
+    if CS.CP.enableGasInterceptor and CS.CP.openpilotLongitudinalControl and enabled:
```

Comment:
It's fine to make the `interceptor_gas_cmd`. You want to gate the sending on line 103